### ISSUE TITLE
Templates: Move "Related Features" tab into "Sampling Features"

### ISSUE THREAD

**Issue by aufdenkampe** [CZOData/YODA-File#47]
I think that for all of our presently targeted templates (Time Series, Specimen Time Series and Specimens, see [excel_templates](https://github.com/ODM2/YODA-File/tree/master/excel_templates)) that it makes sense to limit the cardinality of Related Features so that each Sampling Feature can only have one parent. Although we designed ODM2 to handle compound specimens with multiple parents, we can address these and other special use cases with other future templates.

If we do this, then we can move columns B to J from the Related Features tab to the end of the Sampling Features tab in all three of these Excel templates. We would then eliminate the Related Features tab and substantially simplify both the data entry (see issues #26 & #30) and also the code for YODA file generation, especially if we generate YODA files with a single output line for all Sampling Features related metadata, by also appending all Site and Specimen metadata to the same line.

**Comment by SRGDamia1** [comment]
In v0.3.2 of the TimeSeries template, @horsburgh has removed the related features tab.

**Comment by horsburgh** [comment]
Yes - I did remove it because I don't think we need it for that particular template.  Related features will still probably be required for some of the others (maybe), but I just decided to get rid of it to simplify the Time Series Template.
